tests/unit/test_simple.py

```diff
@@ -0,0 +1,26 @@
+from myproxy.client import MyProxyClient
+
+
+def test_members():
+    """Test public method members of MyProxyClient."""
+    expected_members = [
+        'caCertDir', 'changePassphrase', 'destroy', 'getDelegation',
+        'getTrustRoots', 'hostname', 'info', 'locateClientCredentials',
+        'logon', 'openSSLConfFilePath', 'openSSLConfig', 'parseConfig',
+        'port', 'proxyCertLifetime', 'proxyCertMaxLifetime', 'put',
+        'readProxyFile', 'serverDN', 'setDefaultCACertDir',
+        'ssl_verification', 'store', 'writeProxyFile'
+    ]
+    actual_members = dir(MyProxyClient)
+    for member in expected_members:
+        assert member in actual_members
+
+
+def test_simple():
+    """Test a simple instance of MyProxyClient."""
+    hostname = "www.google.com"
```

Comment:
It would be a bit nicer to use an example service, e.g. example.com instead of a random actual website

Reply:
imaginary domain now in, bud